Program.cs

```diff
@@ -1,33 +1,25 @@
-using System;
-using System.Runtime.InteropServices.JavaScript;
-using System.Threading.Tasks;
+namespace TodoMVC;
 
-namespace TodoMVC
+public partial class MainJs
 {
-    public partial class MainJS
-    {
-        static Controller? controller;
+    private static Controller? _controller;
 
-        public static async Task Main()
+    public static async Task Main()
+    {
+        if (!OperatingSystem.IsBrowser())
         {
-            if (!OperatingSystem.IsBrowser())
-            {
-                throw new PlatformNotSupportedException("This demo is expected to run on browser platform");
-            }
-
-            await JSHost.ImportAsync("todoMVC/store.js", "./store.js");
-            await JSHost.ImportAsync("todoMVC/view.js", "./view.js");
-
-            var store = new Store();
-            var view = new View(new Template());
-            controller = new Controller(store, view);
-            Console.WriteLine("Ready!");
+            throw new PlatformNotSupportedException("This demo is expected to run on browser platform");
         }
 
-        [JSExport]
-        public static void OnHashchange(string url)
-        {
-            controller?.SetView(url);
-        }
+        await JSHost.ImportAsync("todoMVC/store.js", "./store.js");
+        await JSHost.ImportAsync("todoMVC/view.js", "./view.js");
+
+        var store = new Store();
+        var view = new View(new Template());
+        _controller = new Controller(store, view);
+        Console.WriteLine("Ready!");
     }
-}
\ No newline at end of file
+
+    [JSExport]
+    public static void OnHashchange(string url) => _controller?.SetView(url);
```

Comment:
I think there was a problem with arrow function JSExport in RC1, but I'm not sure. Does it work with RC1 ?

Reply:
Not sure, I tested this on RC2.

Reply:
Let's not make that change please